src/client_comms.js

```diff
@@ -39,8 +39,12 @@ class ClientComms {
     this.io = new Server(
       expressServer,
       {
-        pingInterval: PING_INTERVAL_MS,
-        pingTimeout: PING_TIMEOUT_MS
+        cors: {
```

Comment:
Do you have a Use Case for leaving this CORS configuration in the production code OR is this purely for one particular development Use Case?

Reply:
mostly development. Unless you plan for students to customize html for a UI without building images. it would allow any custom UI to connect to the robot.